com.microsoft.mrtk.core/Editor/Inspectors/PropertyDrawers/DrawIfPropertyDrawer.cs

```diff
@@ -33,7 +33,7 @@ public override float GetPropertyHeight(SerializedProperty property, GUIContent
                 return 0f;
             }
 
-            return base.GetPropertyHeight(property, label);
```

Comment:
Is not calling into the `base` implementation okay here?

Reply:
Yeah, base will only return the height of the property without accounting for the children (i.e, a single line + field for most assets), while using the static method provides options for also including the children.